docs/en-US/adding-a-new-site/index.md

```diff
@@ -40,13 +40,15 @@ Read here for [more information about Domains and hosts](custom-domains-hosts.md
 
 Now that VVV knows about our site with the name `vvvtest`, it's going to look inside the `www/vvvtest` folder for our site. We need to create and fill this folder. If I had named the site `testables`, the folder would be `www/testables`.
 
-After creating the folder, download a copy of the site into that folder, and create a `provision` subfolder.
+After creating the folder, create a `provision` subfolder, e.g. `www/testables/provision`. See more below about what goes in this folder.
 
 If you'd like to change the folders VVV uses, [read here for more information](custom-paths-and-folders.md)
 
-### Bonus: Git
+### Copying In Site Files
 
-Rather than manually copying files into the folder, copy them into a git repository, and use the `repo` key to tell VVV where to find it.
+If you have an existing site you want to work with, you will be able to manually copy the site files into the `public_html` subdirectory created when the site is provisioned.
```

Comment:
Works for me, but maybe good to clarify what "the folder" references in case it's not clear.